### PR TITLE
Wallet: Remove NetworkType Enum (Regtest, Signet, Testnet, ...)

### PR DESCRIPTION
**Pull request by alvasw**
It doesn't matter on which network Bisq is running. The code works the same everywhere.

### CODE REVIEW

File: wallets/src/main/java/bisq/wallets/bitcoind/BitcoindProcess.java [hunk 1]
```diff
@@ -53,12 +52,11 @@ public BitcoindProcess(RpcConfig rpcConfig, Path dataDir) {
 
     @Override
     public ProcessConfig createProcessConfig() {
-        String networkArg = getParamForNetworkType(rpcConfig.networkType());
         int zmqPort = NetworkUtils.findFreeSystemPort();
         return new ProcessConfig(
                 "bitcoind",
                 List.of(
-                        networkArg,
+                        "-regtest",
```
Comment (chimp1984) [inline wallets/src/main/java/bisq/wallets/bitcoind/BitcoindProcess.java:59]: Is that correct?

File: wallets/src/main/java/bisq/wallets/elementsd/process/ElementsdProcess.java [hunk 2]
```diff
@@ -49,7 +48,7 @@ public ProcessConfig createProcessConfig() {
         return new ProcessConfig(
                 "elementsd",
                 List.of(
-                        "-chain=" + getNetworkName(),
+                        "-chain=elementsregtest",
```
Comment (chimp1984) [inline wallets/src/main/java/bisq/wallets/elementsd/process/ElementsdProcess.java:50]: Is that correct?

**Review by chimp1984 — CHANGES_REQUESTED** [review]

File: wallets/src/main/java/bisq/wallets/bitcoind/BitcoindProcess.java (hunk 1 above)
Comment (alvasw) [reply wallets/src/main/java/bisq/wallets/bitcoind/BitcoindProcess.java:59]: Yes, that's correct but you're right. It's confusing. I renamed it to `BitcoindRegtestProcess` and moved it into the tests module.

File: wallets/src/main/java/bisq/wallets/elementsd/process/ElementsdProcess.java (hunk 2 above)
Comment (alvasw) [reply wallets/src/main/java/bisq/wallets/elementsd/process/ElementsdProcess.java:50]: Yes, that's correct but you're right. It's confusing. I renamed it to `ElementsdRegtestProcess` and moved it into the tests module.

File: wallets/src/main/java/bisq/wallets/bitcoind/BitcoindProcess.java (hunk 1 above)
Comment (chimp1984) [reply wallets/src/main/java/bisq/wallets/bitcoind/BitcoindProcess.java:59]: Ah thanks.

**Review by chimp1984 — APPROVED** [review]
utACK

**Pull request merged**